org.gnome.ghex.json

```diff
@@ -0,0 +1,30 @@
+{
+    "app-id": "org.gnome.ghex",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "ghex",
+    "rename-appdata-file": "ghex.appdata.xml",
+    "rename-desktop-file": "ghex.desktop",
+    "rename-icon": "ghex",
+    "finish-args": [
+        /* X11 + XShm */
+        "--share=ipc", "--socket=x11",
+        /* Wayland */
+        "--socket=wayland",
+        /* Filesystem */
+        "--filesystem=host",
+        /* dconf */
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
I think you need a little bit more for dconf, see here: https://github.com/flathub/org.gnome.iagno/blob/3c77a388f309ce0df25f12d23463a95edd3279c5/org.gnome.iagno.json#L18